to_do_app/lib/detailTask/detailTask.dart

```diff
@@ -0,0 +1,52 @@
+import 'package:auto_route/auto_route.dart';
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+import 'package:to_do_app/domain/card_to_do.dart';
+
+import '../repository/repository_task.dart';
+
+class DetailOfTask extends StatelessWidget {
+  DetailOfTask({
+    Key? key,
+    required this.index,
+  }) : super(key: key);
+
+  final int index;
+  final repository = RepositoryTask();
+
+  @override
+  Widget build(BuildContext context) {
+    final elements = repository.getTasks();
+
+    var element = elements[index];
```

Comment:
I think this can be done outside the build method, think that if this build is called again for some reason and the list was updated and added a item before the index then it would change the entire state of the widget